### PR TITLE
i mapped things again :ohno:

### PR DESCRIPTION
**Pull request by Shnupbups**
Probably overlaps with other PRs, tbh I haven't checked.

### CODE REVIEW

**Comment by Shnupbups** [comment]
OK, I'll yield to #1269

File: mappings/net/minecraft/entity/passive/horse/HorseColors.mapping [hunk 1]
```diff
@@ -0,0 +1,8 @@
+CLASS net/minecraft/class_5149 net/minecraft/entity/passive/horse/HorseColors
```
Comment (liach) [inline mappings/net/minecraft/entity/passive/horse/HorseColors.mapping:1]: I think we use singular names for enums

File: mappings/net/minecraft/world/gen/feature/DeltaFeatureConfig.mapping [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+		ARG 3 minimumRadius
+		ARG 4 maximumRadius
+		ARG 5 maximumRim
+	METHOD method_27114 deserialize (Lcom/mojang/datafixers/Dynamic;)Lnet/minecraft/class_5158;
+	CLASS class_5159 Builder
+		FIELD field_23900 contents Ljava/util/Optional;
+		FIELD field_23901 rim Ljava/util/Optional;
+		FIELD field_23902 minimumRadius I
+		FIELD field_23903 maximumRadius I
+		FIELD field_23904 maximumRim I
```
Comment (liach) [inline mappings/net/minecraft/world/gen/feature/DeltaFeatureConfig.mapping:19]: `min` `max` should suffice

**Comment by liach** [comment]
Marking as refactor as this one moved horse classes to a new subpackage

**Comment by Shnupbups** [comment]
Probably gonna remove all the horse stuff as it's covered by #1269

**Comment by Shnupbups** [comment]
in fact, i'm just gonna close this PR and open a new one without any of the stuff in #1269

**Pull request closed without merging**